### PR TITLE
Reportes

### PR DESCRIPTION
**Pull request by jeffny2015**
Pull Request Reportess

### CODE REVIEW

File: .rake_tasks~ [hunk 1]
```diff
@@ -0,0 +1,56 @@
+about
```
Comment (aleks279) [inline .rake_tasks~]: remove this file. add it in the `.gitignore`

File: app/controllers/reports_controller.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  def update
+  end
+
+  def destroy
+  end
+
+  def report
+  end
+end
```
Comment (aleks279) [inline app/controllers/reports_controller.rb]: missing new line at the end of the file

File: app/controllers/reports_controller.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    end
+
+    @date_inicio = 1
+    @ganancia_por_miembro = @ganancia_total / @cantidad_vendedores
+
+    respond_to do |format|
+      format.html
+      format.pdf do
+        render template: 'reports/_pdf.html',
+        :pdf => "Reportes", # pdf will download as Reportes.pdf
```
Comment (aleks279) [inline app/controllers/reports_controller.rb]: include the month/year in the report name:
`:pdf => "Reporte #{month}-#{year}"`

Comment (aleks279) [inline app/controllers/reports_controller.rb]: User this syntax instead:
```
pdf: "name",
show_as_html: params[:debug].present?
```

**Comment by jeffny2015** [comment]
Merge !!!!

**Review by aleks279 — APPROVED** [review]

**Pull request merged**